docs/en/latest/how-to-build.md

```diff
@@ -125,7 +125,7 @@ Follow the steps below to install Apache APISIX via the source release package.
   cd apisix-${APISIX_VERSION}
   # Create dependencies
   make deps
-  # Install apisix command
+  # Install apisix command (developer do not need to follow this step)
```

Comment:
I am confused. If it is no need, why not remove it?

Reply:
let's remove it. cc @bin-ya